attestation-evidence-broker/src/ttrpc_server/rpc_server.rs

```diff
@@ -0,0 +1,107 @@
+use anyhow::{anyhow, Result};
+use std::sync::Arc;
+
+use crate::aeb_modules::sev_aeb;
+use crate::device::device::{self, DeviceType};
+use crate::protocols::aeb::aeb::{
+    RetrieveAttestationEvidenceRequest, RetrieveAttestationEvidenceResponse,
+    RetrieveAttestationEvidenceSizeRequest, RetrieveAttestationEvidenceSizeResponse,
+};
+use crate::protocols::aeb::aeb_ttrpc::{create_aeb, Aeb};
+
+struct AEBService {}
+
+pub fn new_server<S: Into<String>>(host: S, port: u16) -> ttrpc::Server {
+    let agent_service = Box::new(AEBService {}) as Box<dyn Aeb + Send + Sync>;
+    let agent_worker = Arc::new(agent_service);
+    let aservice = create_aeb(agent_worker);
+
+    let addr: String = format!("{}:{}", host.into(), port.to_string());
+
+    info!("start server {}", addr);
+
+    ttrpc::Server::new()
+        .bind(addr.as_str())
+        .unwrap()
```

Comment:
Wrapping the return value as `Result<ttrpc::Server>` might eliminate these `unwrap`?